--- Dockerfile.1	2022-05-20 16:04:10.693905933 +0000
+++ Dockerfile	2022-05-20 16:07:01.887924669 +0000
@@ -1,4 +1,4 @@
-FROM nimlang/nim:alpine as build
+FROM ghcr.io/maxisoft/nim-docker-images/nim as build
 
 ARG  REPO=https://github.com/zedeus/nitter.git
 
@@ -18,8 +18,11 @@
 &&  nimble install -y --depsOnly \
 &&  nimble build -y -d:release --passC:"-flto" --passL:"-flto" \
 &&  strip -s nitter \
-&&  nimble scss \
-&&  nimble md
+&&  nimble scss
+
+COPY ./about.md /build/public/md/about.md
+
+RUN nimble md
 
 # ---------------------------------------------------------------------
@@ -30,7 +30,7 @@
-ENV  REDIS_HOST="localhost" \
+ENV  REDIS_HOST="redis" \
-      REDIS_PASS="" \
+      REDIS_PASS="moneyprintergobrrr" \
      REDIS_PORT=6379 \
      NITTER_HTTPS="false" \
      NITTER_HOST="nitter.net" \
      NITTER_NAME="nitter" \
-      NITTER_THEME="Nitter"
+      NITTER_THEME="auto_(twitter)" \
      REPLACE_TWITTER="nitter.net" \
      REPLACE_YOUTUBE="piped.kavin.rocks" \
      REPLACE_REDDIT="teddit.net" \
      REPLACE_INSTAGRAM=""

@@ -51,8 +51,7 @@
-&&  apk add --no-cache tini curl pcre su-exec
+&&  apk add --no-cache curl pcre su-exec

@@ -58,7 @@
-HEALTHCHECK CMD curl --fail http://localhost:8080 || exit 1

@@ -60,7 +60,7 @@
-ENTRYPOINT ["/sbin/tini", "--", "/entrypoint.sh"]
+ENTRYPOINT ["/entrypoint.sh"]